presto-main/src/main/java/com/facebook/presto/SystemSessionProperties.java

```diff
@@ -179,6 +180,11 @@ public SystemSessionProperties(
                         "Number of partitions for distributed joins and aggregations",
                         queryManagerConfig.getHashPartitionCount(),
                         false),
+                stringProperty(
+                        PARTITIONING_PROVIDER,
+                        "Partitioning provider. Possible values are: system, <catalog>",
```

Comment:
Same here, I am wondering if we say `Partitioning provider catalog`, it might be self-explained (it's a catalog name to be provided) and we don't' have to give possible values?